src/types/connector.ts

```diff
@@ -0,0 +1,11 @@
+import { InstantSearch } from './instantsearch';
+
+export type RenderOptions<T = unknown> = {
```

Comment:
we haven't decided on this yet, but I'd love to ban 1-letter generics, let's discuss this another time when we find a eslint rule for it